src/themes/dark-theme.less

```diff
@@ -1,22 +0,0 @@
-@import "~antd/lib/style/color/colorPalette.less";
```

Comment:
Where do we have our dark/light themes defined? feels like a clean thing to have in this folder.

Reply:
it is, but these are for ant design not material. i intend to build out and define our theme definitions in this folder once i have the overrides written